docs/source/conf.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+#
+# Open-eObs documentation build configuration file, created by
+# sphinx-quickstart on Wed Oct 18 18:35:36 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import requests
+import zipfile
+import StringIO
+import re
+from shutil import copytree, ignore_patterns, rmtree
+
+
+repo_path = os.path.abspath('../../')
+sys.path.insert(0, repo_path)
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+    'sphinx.ext.doctest',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.coverage',
+    'sphinx.ext.viewcode',
+    'sphinx.ext.githubpages',
+    'sphinxodoo.ext.autodoc'
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
```

Comment:
Just add a brief comment here explaining why this was necessary. Will help in future when we need to make a change.